examples/ampcontext-creative.js

```diff
@@ -0,0 +1,88 @@
+if (!window.context){
+  // window.context doesn't exist yet, must perform steps to create it
+  // before using it
+  console.log("window.context NOT READY");
+
+  // must add listener for the creation of window.context
+  window.addEventListener('amp-windowContextCreated', function(){
+    console.log("window.context created and ready to use");
+    window.context.onResizeSuccess(resizeSuccessCallback);
+    window.context.onResizeDenied(resizeDeniedCallback);
+  });
+
+  // load ampcontext-lib.js which will create window.context
+  ampContextScript = document.createElement('script');
+  ampContextScript.src = "../dist.3p/current/ampcontext-lib.js";
```

Comment:
So this example is only for use in local dev mode?  Maybe worth documenting at the top.  (Come to think of it, a short comment block at the top about the purpose of the example would also be useful.  Plus the AMP copyright boilerplate.)